src/org/labkey/test/util/AuditLogHelper.java

```diff
@@ -69,6 +80,117 @@ public DataRegionTable goToAuditEventView(String eventType)
         return new DataRegionTable("query", _wrapper);
     }
 
+    /**
+     * Get the audit logs from the LabKey server filtered to the given project.
+     *
+     * @param containerPath Path of the LK container to use for the select command.
+     * @param auditEventName Name of the audit event to filter on. Example 'SamplesWorkflowAuditEvent'.
+     * @param columnNames The name of the columns to return.
+     * @param filters The filters to be applied
+     * @return A rowResponse with the query logs.
+     * @throws IOException Can be thrown by the SelectRowsCommand.
+     * @throws CommandException Can be thrown by the SelectRowsCommand.
+     */
+    public SelectRowsResponse getAuditLogsFromLKS(String containerPath, String auditEventName, List<String> columnNames,
+                                                         List<Filter> filters, @Nullable Integer maxRows) throws IOException, CommandException
+    {
+        SelectRowsCommand cmd = new SelectRowsCommand("auditLog", auditEventName);
+        cmd.setColumns(columnNames);
+        cmd.addFilter("ProjectId/Name", _wrapper.getCurrentProject(), Filter.Operator.EQUAL);
+        filters.forEach(cmd::addFilter);
+        if (maxRows != null)
+            cmd.setMaxRows(maxRows);
+        if (!containerPath.startsWith("/"))
+            containerPath = "/" + containerPath;
+        return cmd.execute(WebTestHelper.getRemoteApiConnection(), containerPath);
```

Comment:
For consistency, use the connection supplier in this class rather than the `WebTestHelper` one.
```suggestion
        return cmd.execute(_connectionSupplier.get(), containerPath);
```

Reply:
Done